src/cpu/core_dyn_x86/cache.h

```diff
@@ -487,19 +487,22 @@ static INLINE void cache_addb(Bit8u val) {
 	*cache.pos++=val;
 }
 
-static INLINE void cache_addw(Bit16u val) {
-	*(Bit16u*)cache.pos=val;
-	cache.pos+=2;
+static INLINE void cache_addw(const uint16_t &val)
```

Comment:
*Just a comment, the code is fine*

I think we might be abusing `INLINE` macro… maybe because it's named INLINE and not FORCE_INLINE - I'm not sure... In this particular case it seems to make sense, but in the future we'll need to make some measurements… and perhaps I will need to describe in contributing guidelines, that usual C++ `inline` is not the same as this macro.

Reply:
Indeed; we could try knocking out the inlines and see if it changes anything (leave it up to gcc to decide).  I also need to change these cache functions to pass by value.